### PR TITLE
Feature/smarteole

### PR DESCRIPTION
**Pull request by paulf81**
Not yet ready to merge

**Feature or improvement description**
This is the rearrangement of the smarteole examples as an example folder within FLASC

**Impacted areas of the software**
Creates a new example folder examples_smarteole

### CODE REVIEW

**Review by ejsimley — COMMENTED** [review]
One other comment for now is that I prefer changing the "examples" folder to something more descriptive, like "examples_artificial_dataset" to distinguish it from "examples_smarteole"

File: examples_smarteole/01_precalculate_floris_solutions.py
[the remark this answers is not in the document]
Comment (paulf81) [reply examples_smarteole/01_precalculate_floris_solutions.py:1]: on it!

**Comment by paulf81** [comment]
ok @ejsimley balls in your court!

**Comment by ejsimley** [comment]
@paulf81, I reviewed and updated all the smarteole example notebooks. See how the commentary and everything looks in the notebooks. I also updated the energy ratio gain plotting method to use "Change in energy ratio" for the y-axis label. 

The last thing is that I suggest renaming the "examples" folder. Since we have "examples_smarteole" I think it should be something like "examples_artificial_data". Do you agree, and if so, do you want to pick a name for that? 

And then just a reminder that the FLORIS Version in requirements and setup.py will need to be updated before the release.

**Comment by paulf81** [comment]
I like "examples_artificial_data", happy to make the change, just a quick check @misi9170 / @ejsimley does that name work for you?

**Comment by ejsimley** [comment]
Hi @paulf81, that power curve plot from example 03 looks reasonable to me. We found that there was a nacelle transfer function we had to apply to the SCADA wind speed measurements in the paper (which we determined using nacelle lidar measurements), which is why the curve looks a little different than the manufacturer's power curve. But I think that's ok for this example.

**Comment by paulf81** [comment]
> Hi @paulf81, that power curve plot from example 03 looks reasonable to me. We found that there was a nacelle transfer function we had to apply to the SCADA wind speed measurements in the paper (which we determined using nacelle lidar measurements), which is why the curve looks a little different than the manufacturer's power curve. But I think that's ok for this example.

Ok sounds good, I was thinking that the uncertainty bounds seem to run all the way down to 0, but maybe I'm misreading the plot?

**Comment by paulf81** [comment]
ok!  I made the change in the example folder and maybe if we just let this issue of the power curve coloring alone, I think we're ready to merge and close the book on this one?

**Pull request merged**